tests/features/paragraphs/custom_page.paragraphs.feature

```diff
@@ -20,25 +20,25 @@ Feature:
     When I open the header local tasks menu
     And I click "Edit"
     # The first paragraphs item is open by default.
-    Then there should be 1 row in the "Custom page body" field
-    Then I should see the "Remove" button in the "Custom page body" field at row 1
+    Then there should be 1 paragraph in the "Custom page body" field
+    Then I should see the "Remove" button in the "Custom page body" field for paragraph 1
```

Comment:
So, we are not going with the oe_paragraphs in the end? or what?

Reply:
No since you advised against it. See my comment above.